Sprint-1/interpret/time-format.js

```diff
@@ -11,14 +11,31 @@ console.log(result);
 
 // For the piece of code above, read the code and then answer the following questions
 
-// a) How many variable declarations are there in this program?
+// a) How many variable declarations are there in this program? 6
 
-// b) How many function calls are there?
+// b) How many function calls are there? 1
 
 // c) Using documentation, explain what the expression movieLength % 60 represents
+/*it means dividing 8784 by 60 and output the remainder value of this division. */
+
 
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
+/*It calculates the movie length in minutes */
```

Comment:
For (c) and (d), I wonder how ChatGPT would explain the code.  (I know explaining something this simple or obvious can be challenging)